examples/README.md

```diff
@@ -0,0 +1,29 @@
+# Examples for Testing Features Locally
+
+The example projects are meant to be used to test features locally by contributors working on this SDK.
+
+## Steps to Test Your Code
+
+If you are contributing changes to this SDK, you can test those changes by using the `prerecorded`, `streaming`, or `manage` "hello world"-style applications in the `examples` folder. Here are the steps to follow:
+
+### Set your API Key as an Environment Variable named "DEEPGRAM_API_KEY"
+
+If using bash, this could be done in your `.bash_profile` like so:
+
+```bash
+export DEEPGRAM_API_KEY = "YOUR_DEEPGRAM_API_KEY"
+```
+
+or this could also be done by a simple export before executing your python application:
+
+```bash
+DEEPGRAM_API_KEY="YOUR_DEEPGRAM_API_KEY" python main.py
+```
+
+### Run the project
+
+Make sure you're in the directory with the `main.go` file and run the project with the following command.
```

Comment:
need to change to python

Reply:
ooops I forgot to save the file as second time before committing it. should be fixed now.